opentreemap/opentreemap/celery.py

```diff
@@ -16,3 +19,12 @@
 # pickle the object when using Windows.
 app.config_from_object('django.conf:settings')
 app.autodiscover_tasks(lambda: settings.INSTALLED_APPS)
+
+
+@task_failure.connect
+def handle_task_failure(**kw):
+    rollbar_settings = getattr(settings, 'ROLLBAR', {})
+    access_token = rollbar_settings.get('access_token')
+    if access_token:
+        rollbar.init(access_token, rollbar_settings['environment'])
```

Comment:
This handler will get called for every exception. Did you run across any documentation regarding whether or not it is a good idea to call `init` multiple times?